### PR TITLE
:memo: add navigation in core package

### PR DESCRIPTION
**Pull request by fabienjuif**
https://github.com/alakarteio/k-ramel/blob/doc/navigation/packages/k-ramel/doc/LISTENERS.md

The doc will be under this two navigation

### CODE REVIEW

File: packages/k-ramel/README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-TODO
+# k-ramel
+
+State manager for your components apps, the safe and easy way.
```
Comment (bpetetot) [inline packages/k-ramel/README.md]: State manager for your **app components**, the safe and easy way.

File: packages/k-ramel/README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+  <img src="./doc/logo.png" width="400" />
+</p>
+
+## Why should you give it a try ? 🤔
+Because `k-ramel`:
+ - ⚡️ is fast
+ - 📸 is immutable
+ - 📦 is modular
+ - 💎 encourages to decouple UI and state management
+ - 💥 encourages to not have side effect into your buisiness logic
```
Comment (bpetetot) [inline packages/k-ramel/README.md]: encourages to not have side effect into your **business logic**

File: packages/k-ramel/doc/DRIVERS.md [hunk 3]
```diff
@@ -1 +1,25 @@
-TODO
+# Drivers
+
+<p align="center">
```
Comment (delphinemillet) [inline packages/k-ramel/doc/DRIVERS.md]: could be a title

[the remark this answers is not in the document]
Comment (fabienjuif) [reply packages/k-ramel/doc/DRIVERS.md]: _main navigation_ [<a href="../../../README.md#how-to-use-k-ramel">ecosystem documentation</a>]&nbsp;[<a href="../README.md#how-to-use-k-ramel">core documentation</a>]&nbsp;[<a href="../../../README.md#modules">modules</a>]

_sub navigation_ <<a href="./LISTENERS.md">listeners</a>&nbsp;|&nbsp; <a href="./REACTIONS.md">reactions</a>&nbsp;|&nbsp; <a href="./DRIVERS.md">drivers</a>&nbsp;|&nbsp; <a href="./STORE.md">store</a>>

_sub navigation_ [<a href="./LISTENERS.md">listeners</a>&nbsp;|&nbsp; <a href="./REACTIONS.md">reactions</a>&nbsp;|&nbsp; <a href="./DRIVERS.md">drivers</a>&nbsp;|&nbsp; <a href="./STORE.md">store</a>]

[the remark this answers is not in the document]
Comment (delphinemillet) [reply packages/k-ramel/doc/DRIVERS.md]: I don't have strong opinion here 😅

**Pull request merged**